sys/parsoid.js

```diff
@@ -621,9 +621,7 @@ PSP.transformRevision = function(hyper, req, from, to) {
             original: original
         };
         if (from === 'sections') {
-            body2.html = {
-                body: replaceSections(original, parseSections(req))
-            };
+            body2.html = replaceSections(original, parseSections(req));
```

Comment:
@arlolra, the bit I was missing was this:

> They continue to pass because Parsoid accepts the both framed and unframed html.

This makes sense now.

As a general feedback, it might be helpful to provide a bit more context in the commit message, so that the intention & background of the changes is clearer during review. You were (again) very patient with me, but I think it could save us both some time if things were clearer up-front.